home-server-generator/pom.xml

```diff
@@ -13,7 +13,7 @@
 
     <properties>
         <yaml.name>openapi.yaml</yaml.name>
-        <generation.files>OffsetDateTimeProvider.java,pom.xml</generation.files>
+        <generation.files>pom.xml</generation.files>
```

Comment:
What would be if we just generate all supporting files now?

Reply:
Not sure if I understood what you mean. If you are about Offset, we discussed that we don't need generation time so pom.xml only left. Without generation files doesn't work at all because I have to specify newer maven plugin in pom.

Reply:
Please look at the doc https://openapi-generator.tech/docs/generators/jaxrs-spec. There is no such option as  `supportingFilesToGenerate`. Please remove.

Reply:
Deleted.